composer/modules/server/services/ballerina-parser/src/main/java/org/ballerinalang/composer/service/ballerina/parser/service/JSONModelConstants.java

```diff
@@ -0,0 +1,49 @@
+/*
+*  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Invalid License Header

Reply:
It is a rename of the file "BLangJSONModelConstants.java" into "JSONModelConstants.java" along with some edits. I suppose we can keep the license header as it is.